src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,610 @@
+| block_number_minted  | integer          | The block height when the NFT was minted.                                                        |
+| block_number         | integer          | The block height when the amount or owner changed.                                               |
+| chain                | string           | Chain name. One of `AVALANCHE`, `BSC`, `ETH`, `FANTOM`, or `POLYGON`.                            |
+| collection_name      | string           | May be `null`. An NFT collection name.                                                                          |
+| contract_type        | string           | The type of NFT contract standard. One of `ERC721` or `ERC1155`.                                 |
+| last_token_uri_sync  | string           | When the token_uri was last updated.                                                             |
+| last_metadata_sync   | string           | When the metadata was last updated.                                                              |
+| metadata             | string           | The metadata of the token.                                                                       |
+| minter_address       | string           | Minter address.                                                                                  |
+| owner_of             | string           | The wallet address of the owner of the NFT.                                                      |
+| possible_spam        | boolean          | May be `null`. If `true`, the contract address is has [been identified](https://docs.moralis.io/web3-data-api/evm/nft-spam-detection) as associated with spam, phishing attempts, or suspicious activities. |
+| symbol               | string           | May be `null`. The symbol of the NFT contract.                                                                  |
+| token_address        | string           | The address of the NFT contract.                                                                 |
+| token_id             | string           | The token ID of the NFT.                                                                         |
+| token_hash           | string           | The token hash.                                                                                  |
+| token_uri            | string           | The URI to the metadata of the token.                                                            |
+| uri_meta             | object           | A TokenMetadata object. The contents of this object varies, but will include values for `image`, `token_name`, `description`, `attributes` or `animation_url` (which might be `null`). |
+
+
+<DevComment>
+For uri_meta, I think only the values under the `attributes` key are variable. The other keys are always present, but may be `null`. This should link to a standard object.
```

Comment:
well as we use Json object as the type for attributes it can have different view.
```rust
pub(crate) struct UriMeta {
    pub(crate) image: Option<String>,
    #[serde(rename(deserialize = "name"))]
    pub(crate) token_name: Option<String>,
    description: Option<String>,
    attributes: Option<Json>,
    animation_url: Option<String>,
}
```
Actually fields for uri_meta are collected from link from token_uri parameter.

Form [this example](https://unft-metadata.rgc-external-stage.unioverse.com/getMetadataForCollectionAndTokenV3/Unioverse%20Collectibles/1265789) attributes is just one object, but [here](https://tikimetadata.s3.amazonaws.com/tiki_box.json) it is the array. So we decided just to try to parse the whole Json for attributes. bcz nft authors can add there any data.

Reply:
Update to include a standard TokenMetadata object